docs/verbs/catkin_build.rst

```diff
@@ -480,8 +480,20 @@ To run catkin tests for catkin packages, use the following:
 
 .. code-block:: bash
 
+    $ catkin run_tests --verbose -i
```

Comment:
I took this from my notes last time i was running tests - I think I needed the `-i` to show me why my tests were failing